docs/performance/inserts/parallel.txt

```diff
@@ -0,0 +1,130 @@
+.. _parallel_inserts:
+
+================
+Parallel Inserts
+================
+
+Because CrateDB is a distributed database with every node capable of handling
+client requests, parallelizing your inserts (i.e. doing many at once) is a
```

Comment:
Maybe we could also explain why that's a good approach?
E.g. mention that users have to make sure that the client isn't the bottle neck. E.g. that if there is a latency of 5ms between client and server, a client doing sequential inserts couldn't reach more than 200 inserts/s even if the insert on the server side required 0ms processing time. Concurrent or parallel inserts can help there (there is a difference between concurreny and parallelism, though it's often used interchangeable..)

Reply:
should we rename this section to "concurrent requests"? and mention the parallel difference. or the other way around?

Reply:
I'd rename it to concurrent inserts - I wouldn't mention the parallel difference. But it might make sense to explain the latency aspect.

Reply:
done